Packs/CortexXDR/ReleaseNotes/4_9_4.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cortex XDR - IOC
+- Added `the xsoar_comments_field` parameter.
```

Comment:
```suggestion
- Added the *xsoar_comments_field* integration parameter.
```

It would be helpful for customers if you could provide more information about this parameter's purpose.

Reply:
I started describing it, but it became a bit too long for a release note (it's a bit complex), so I decided to keep it in the `description` part of the yaml instead. wdyt?

Reply:
What about something like this?:
```suggestion
- Added `the xsoar_comments_field` parameter to add tags to XDR IOCs.
```

Reply:
perhaps 
`- Added `the xsoar_comments_field` parameter, to choose the field used when pushing comments to XDR. The default remains *comments*.`